r/R/dplyr.R

```diff
@@ -46,7 +46,12 @@ arrow_dplyr_query <- function(.data) {
       # drop_empty_groups is a logical value indicating whether to drop
       # groups formed by factor levels that don't appear in the data. It
       # should be non-null only when the data is grouped.
-      drop_empty_groups = NULL
+      drop_empty_groups = NULL,
+      # arrange_vars will be a list of expressions
+      arrange_vars = list(),
```

Comment:
Why isn't this a character vector like `group_by_vars`?

Reply:
`dplyr::arrange()` supports sorting by expressions, not just columns, so I designed this to support that. It just needs to be wired up in the `collect` method which I'm still workking out how best to do.